### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by yangxin9003**
完善docs
* 基础           
  * [x] 安装
  * [x] App
  * [ ] Grid
  * [x] Flex
  * [x] Button

* 表单         
  * [x] Input
  * [x] Select
  * [x] Checkbox
  * [x] Radio
  * [x] Form
  * [x] DatePicker
  * [ ] Upload

* 区块
  * [ ] Panel
  * [ ] Table
  * [ ] Tabs
  * [ ] Pagination
  * [ ] Dropdown
  * [ ] Modal
  * [ ] DragModal
  * [ ] Menu
  * [ ] Tag
  * [ ] tree
* 消息
  * [ ] Message
  * [ ] Tooltip
  * [ ] pop-confirm")|PopConfirm

### CODE REVIEW

File: src/components/flex/_flex.js [hunk 1]
```diff
@@ -57,7 +57,6 @@ const AuFlex = Vue.extend({
           style['align-items'] = 'center'
         }
       }
-
```
Comment (lovekonakona) [inline src/components/flex/_flex.js]: 为什么去掉这个空行

File: docs/_layout.jade [hunk 2]
```diff
@@ -211,7 +222,7 @@ html.no-js
           props: {
             html: String
           },
-          mounted () {
+          mounted() {
```
Comment (lovekonakona) [inline docs/_layout.jade]: 为什么去掉了一个空格

File: docs/app.jade [hunk 3]
```diff
@@ -4,29 +4,80 @@ block title
...
-        au-app
-          au-header
-            div(slot="logo")|Hello
-          au-sideber
-            au-menu
-          au-container
-            au-panel
+    .markdown-body
+      :markdown-it
+          # 基础组件
```
Comment (lovekonakona) [inline docs/app.jade]: 缩进不是两个空格?

File: src/components/flex/_flex.js (hunk 1 above)
Comment (yangxin9003) [inline src/components/flex/_flex.js]: src下的内容已回滚

**Pull request merged**